pynsxv/library/libutils.py

```diff
@@ -95,12 +95,13 @@ def get_scope(client_session, transport_zone_name):
     try:
         vdn_scopes = client_session.read('vdnScopes', 'read')['body']
         vdn_scope_list = client_session.normalize_list_return(vdn_scopes['vdnScopes'])
-        vdn_scope = [scope['vdnScope'] for scope in vdn_scope_list
-                     if scope['vdnScope']['name'] == transport_zone_name][0]
+	vdn_scope = [scope['vdnScope'] for scope in vdn_scope_list
+			if scope['vdnScope'][0]['name'] == transport_zone_name][0]
```

Comment:
scope['vdnScope'] is a dict, so scope['vdnScope'][0] will cause a KeyError. This will lead to the function returning None, None all the time, as the try/except block triggers on KeyErrors. With this change the logical switch creation will never work. What was your intend with this change?